37/generated/admin_client_config.html

```diff
@@ -304,7 +304,7 @@ <h4><a id="ssl.protocol"></a><a id="adminclientconfigs_ssl.protocol" href="#admi
 <p>The SSL protocol used to generate the SSLContext. The default is 'TLSv1.3' when running with Java 11 or newer, 'TLSv1.2' otherwise. This value should be fine for most use cases. Allowed values in recent JVMs are 'TLSv1.2' and 'TLSv1.3'. 'TLS', 'TLSv1.1', 'SSL', 'SSLv2' and 'SSLv3' may be supported in older JVMs, but their usage is discouraged due to known security vulnerabilities. With the default value for this config and 'ssl.enabled.protocols', clients will downgrade to 'TLSv1.2' if the server does not support 'TLSv1.3'. If this config is set to 'TLSv1.2', clients will not use 'TLSv1.3' even if it is one of the values in ssl.enabled.protocols and the server only supports 'TLSv1.3'.</p>
 <table><tbody>
 <tr><th>Type:</th><td>string</td></tr>
-<tr><th>Default:</th><td>TLSv1.2</td></tr>
+<tr><th>Default:</th><td>TLSv1.3</td></tr>
```

Comment:
note for reviewers (ore future readers) since I investigated a bit on this. 

It may seem like that we are changing the default here with 3.7. But this default for JDK 11 exists for a very long time (since 2020). The default value of these generated docs vary, depending on what JDK version was being used while generating the docs. It just happens that we used JDK 11 while generating docs here.

Reply:
thanks. I did check that this was supported since 2020 and simply assumed it wasn't synced - interesting catch on the JDK version used to generate it!